app/components/shared/site-navigation.tsx

```diff
@@ -18,7 +18,8 @@ export function SiteNavigation() {
             <span className="text-xl font-bold">ReactJS ID</span>
           </Link>
         </div>
-        <ul className="flex gap-10 text-slate-400">
+        {/* @todo: change xs:hidden with burger menu? */}
```

Comment:
Remove the todo as it will be another issue to improve the responsive mobile design.